libsplinter/src/biome/rest_api/resources/credentials.rs

```diff
@@ -19,3 +19,9 @@ pub(crate) struct UsernamePassword {
     pub username: String,
     pub hashed_password: String,
 }
+
+#[derive(Deserialize, Serialize)]
+pub(crate) struct NewUser {
```

Comment:
Given how it's used - you can probably do this as
```rust
struct NewUser<'a> {
    pub user_id: &'a str,
    pub username: &'a str,
}
```
Plus, it doesn't need to be deserializable , since it is only for output.

Reply:
Addressed.